usage.txt

```diff
@@ -8,3 +8,4 @@ Usage
 
   Optional options:
     out:               path to output the installer (path must exist)
+    nsiTemplate:       path to override the installer.nsi template for win platform (path must exist)
```

Comment:
This should also be handled in inside of the config file, or?
As it is really windows specific. :)